crates/transpiler/src/passes/unitary_synthesis.rs

```diff
@@ -1454,7 +1457,58 @@ fn run_2q_unitary_synthesis(
     Ok(())
 }
 
+#[pyfunction]
+#[pyo3(name = "synthesize_unitary_matrix", signature=(unitary, qubit_indices, target, basis_gates,coupling_edges, approximation_degree=None, natural_direction=None, pulse_optimize=None))]
```

Comment:
I would've expected Rust's formatter to unify this, but maybe the spaces are optional? 🤔 
```suggestion
#[pyo3(name = "synthesize_unitary_matrix", signature=(unitary, qubit_indices, target, basis_gates, coupling_edges, approximation_degree=None, natural_direction=None, pulse_optimize=None))]
```

Reply:
Fixed in [82bd240](https://github.com/Qiskit/qiskit/pull/15265/commits/82bd2406c93e4bd30cd5cd6b1ad6c01922e8d9d0).

Interesting, it seems that I can heavily mess up the `pyo3` declaration without `cargo fmt` complaining.

BTW, we are heavily using both ``#[pyo3(signature=(...)]`` and ``#[pyo3(signature = (...)]`` throughout the code (note the spaces around the equality sign). So don't complain that we have spaces next to `name` but not next to `signature` 😄.